sdk/core/azure-core/azure/core/pipeline/_base.py

```diff
@@ -163,7 +163,7 @@ def _prepare_multipart_mixed_request(request):
 
         Does nothing if "set_multipart_mixed" was never called.
         """
-        multipart_mixed_info = request.multipart_mixed_info  # type: ignore
+        multipart_mixed_info = request.multipart_mixed_info # type: ignore
```

Comment:
All type ignore should be associated with an open issue of some sort. Do you get why there is a problem here? (I know this one is tricky, but I want to challenge you to understand the problem, and define it precisely, and re-challenge the design of azure-core if necessary).